libs/agora/genes/src/lib/components/gene-resources/gene-resources.component.html

```diff
@@ -9,11 +9,11 @@ <h2 id="target-enabling-resources-header">Target Enabling Resources</h2>
             {{ gene.hgnc_symbol || gene.ensembl_gene_id }} that are currently available, under
             development, or planned.
           </p>
-          <div class="row tab-row last-row">
-            <div id="target-enabling-resources-card1" class="col-md-3 col-sm-8 col-8 header-title">
```

Comment:
col-sm* and col-md* are part of the bootstrap dependency that was removed for migration to the monorepo